src/BribeInitiative.sol

```diff
@@ -88,19 +88,19 @@ contract BribeInitiative is IInitiative, IBribeInitiative {
             lqtyAllocationByUserAtEpoch[_user].getItem(_prevLQTYAllocationEpoch);
 
         require(
-            lqtyAllocation.value != 0 && _prevLQTYAllocationEpoch <= _epoch
-                && (lqtyAllocation.next > _epoch || lqtyAllocation.next == 0),
+            _prevLQTYAllocationEpoch <= _epoch && (lqtyAllocation.next > _epoch || lqtyAllocation.next == 0),
             "BribeInitiative: invalid-prev-lqty-allocation-epoch"
         );
         DoubleLinkedList.Item memory totalLQTYAllocation =
             totalLQTYAllocationByEpoch.getItem(_prevTotalLQTYAllocationEpoch);
         require(
-            totalLQTYAllocation.value != 0 && _prevTotalLQTYAllocationEpoch <= _epoch
+            _prevTotalLQTYAllocationEpoch <= _epoch
                 && (totalLQTYAllocation.next > _epoch || totalLQTYAllocation.next == 0),
             "BribeInitiative: invalid-prev-total-lqty-allocation-epoch"
         );
 
         (uint88 totalLQTY, uint120 totalAverageTimestamp) = _decodeLQTYAllocation(totalLQTYAllocation.value);
+        require(totalLQTY > 0, "BribeInitiative: invalid-prev-total-lqty");
```

Comment:
I would rephrase the error message. The variable being checked stands for the total LQTY allocated to the initiative in the epoch from which the bribes are being claimed from. Thus, I guess a better message would be: `invalid-total-lqty-allocation` or `total-lqty-allocation-zero`.